cypress/component/basic/enzyme/README.md

```diff
@@ -0,0 +1,98 @@
+# Enzyme examples
+
+This folder shows several examples from [Enzyme docs](https://enzymejs.github.io/enzyme/).
+
+In general if you are migrating from Enzyme to `cypress-react-unit-test`:
+
+- there is no shallow mounting, only the full mounting. Thus `cypress-react-unit-test` has `mount` which is similar to the Enzyme's `mount`
+- you can mock [children components](https://github.com/bahmutov/cypress-react-unit-test/tree/main/cypress/component/advanced/mocking-component) if you want to avoid running "expensive" components during tests
+- the test is running as a "mini" web application. Thus if you want to set a context around component, then set the [context around the component](https://github.com/bahmutov/cypress-react-unit-test/tree/main/cypress/component/advanced/context)
+
+## setState
+
+If you want to change the component's internal state, use the component reference. You can get it by using the special property `ref` when mounting.
```

Comment:
I would probably avoid making this public – because it is potentially unsafe and makes the our tests know about the internal implementation of the component. When we have a component we must have a way to set any state case using props and UX.

Moreover, it is not possible to use hooks at all. Maybe would be a good decision to rename this method to `dangerous_setState`?

Reply:
yeah, but it comes from React, right, so I say we can show this BUT we don't make it easy. We don't provide this sugar for reference, etc. You _really_ want to use `setState`, here is an example.

Honestly, with all these examples, I was like - I never had to do this to actually write useful component tests, this is just _unit_ testing stuff.